content/nginx-one/secure-your-fleet/secure.md

```diff
@@ -0,0 +1,150 @@
+1. Select **Send email** to confirm.
+1. You should receive a verification code in the email provided. Copy that code.
+1. Under the **Actions** column, select **Enter verification code**.
+1. Paste the code and select **Verify receiver**.
+
+## Configure Alert Policy
+
+Next, configure the policy that identifies when you'll get an alert. You'll need to reference available alerts in our [NGINX One Console Glossary]({{< ref "/nginx-one/glossary.md#nginx-alerts/" >}}). Relevant security alerts include:
+
+- SecurityRecommendationNGINX
+- HighCVENGINX
+- MediumCVENGINX
+- LowCVENGINX
+
+1. Go to **Alerts Management > Alert Policies**.
+1. Select **Add Alert Policy**.
+   1. Enter the name of your choice. You're limited to lower-case characters, numbers, and dashes.
+   1. (Optional) Specify a label and description.
+1. Under **Alert Reciever Configuration > Alert Receivers,** select the **Alert Receiver** you just created.
+1. Under **Policy Rules** select **Configure**.
+1. In the **Policy Rules** screen that appears, select **Add Item**.
+1. In the **Route** window that appears, review the **Select Alerts** drop-down.
+1. Under **Select Alerts** select a filter. Now select **Matching Custom Criteria > Alertname > Configure**. In the screen that appears, use **Exact Match** and copy/paste an alert name from the [NGINX One Console Glossary]({{< ref "/nginx-one/glossary.md#nginx-alerts" >}}).
+1. Select **Apply** to exit the **Alertname** window.
+1. Select **Apply** to exit the **Route** window.
+1. Select **Apply** to exit the **Policy Rules** window.
+1. You can now select the **Add Alert policy** button.
+1. Set the **Action as Send** and select **Apply**.
+
+## Create more alert policies  
+
+Repeat the process described in [Configure Alert Policy](#configure-alert-policy) section. Repeat again if and as needed for all of the alerts in the 
+[NGINX One Console Glossary]({{< ref "/nginx-one/glossary.md#nginx-alerts/" >}}).
+
+## Activate the alert policy
+
+Now to make sure your new policy works, add your new policies to the list of **Active Alert Policies**. To do so:
+
+1. Select **Alerts Management > Active Alert Policies**
+1. Select **Select Active Alert Policies**. 
+1. In the **Select Active Alert Policies** window, select **Add Item**
+1. In the drop-down box that appears, select the Alert Policy that you created. 
+1. Select the **Add Select Active Alert Policies** button.	
+1. Select **Add Item**
+
+You've now set up F5 Distributed Cloud to send you alerts from NGINX One Console, to your email address. When the alert policy identifies an alert, it sends you an email from **alerts@cloud.f5.com**.
+
+## Summary
+
+In this tutorial, you learned how to:
+
+- Access the NGINX One Console
+- Connect an NGINX instance
+- Configure and activate an alert
+
+You will now receive an email any time the F5 Distributed Cloud sees one or more of the alerts that you configued. 
```

Comment:
This repeats the "By the end of this tutorial, you'll be able to: ..." section at the top. Not sure a summary is needed.